crates/pop-cli/src/commands/call/chain.rs

```diff
@@ -776,11 +866,13 @@ mod tests {
 			args: vec![].to_vec(),
 			url: Some(Url::parse(POP_NETWORK_TESTNET_URL)?),
 			suri: None,
+			use_wallet: false,
 			skip_confirm: false,
 			call_data: Some("0x00000411".to_string()),
 			sudo: false,
 		};
 		let mut cli = MockCli::new()
+			.expect_confirm("Do you want to use your browser wallet to sign the transaction? (Selecting 'No' will prompt you to manually enter the secret key URI for signing, e.g., '//Alice')", false)
```

Comment:
```suggestion
			.expect_confirm("Do you want to use your browser wallet to sign the extrinsic? (Selecting 'No' will prompt you to manually enter the secret key URI for signing, e.g., '//Alice')", false)
```

Reply:
Refactored here into wallet::prompt_to_use_wallet: https://github.com/r0gue-io/pop-cli/commit/6035fd86b7ca679127b5cd34765a51dca6a66f13#diff-f1b7ff543f883f3dc2a2c8b3fc1c93e144e63e718655d11893011bc254f32e47R53.

Note: there is a follow-up commit changing to extrinsic instead of transaction.